config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -45,6 +45,7 @@ presubmits:
         - --gcp-zone=us-west1-b
         - --ginkgo-parallel=30
         - --provider=gce
+        - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-storage-slow
```

Comment:
Are we tracking the recreation of this bucket in a community owned GCP project? This bucket is not in a kubernetes.io GCP project.

Reply:
@upodroid looks like we are NOT! - there's tons of references https://cs.k8s.io/?q=gs%3A%2F%2Fkubernetes-release-pull&i=nope&files=&excludeFiles=&repos=